InfiniTAM/infinitam_ros_node.cpp

```diff
@@ -313,109 +312,105 @@ bool InfinitamNode::publishMap(std_srvs::Empty::Request& request,
   nr_triangles = main_engine_->GetMesh()->noTotalTriangles;
-  nr_points = nr_triangles * 3;
-  ROS_INFO_STREAM(
-    "nr_triangles:  " << nr_triangles);ROS_INFO_STREAM(
-    "nr_points:  " << nr_points);
+  nr_points = nr_triangles * 3u;
+  ROS_INFO_STREAM("nr_triangles:  " << nr_triangles);
+  ROS_INFO_STREAM("nr_points:  " << nr_points);
 
   // Build the point cloud.
   pcl::toROSMsg(*point_cloud_pcl, polygon_mesh_ptr->cloud);
 
   // write vertices
   ROS_DEBUG_STREAM("going to fill the mesh with faces.");
 
-mesh_ptr_->polygons.resize(nr_triangles);
+  mesh_ptr_->polygons.resize(nr_triangles);
 
-  for (uint i = 0; i < nr_triangles; ++i) {
+  for (std::size_t i = 0u; i < nr_triangles; ++i) {
     //  Write faces.
-    mesh_ptr_->polygons[i].vertices.resize(3);
+    mesh_ptr_->polygons[i].vertices.resize(3u);
     // The vertex index starts with 1 not with 0 (OBJ-file standard).
     // the Obj_io.h defines that the vertices start with 0, when writing a file,
     // it just adds one to each value, so for publishing:
-    // polygon_mesh_ptr->polygons[i].vertices.push_back(i * 3 + 2 + 1);
-    // polygon_mesh_ptr->polygons[i].vertices.push_back(i * 3 + 1 + 1);
-    // polygon_mesh_ptr->polygons[i].vertices.push_back(i * 3 + 0 + 1);
+    polygon_mesh_ptr->polygons[i].vertices.push_back(i * 3u + 2u + 1u);
+    polygon_mesh_ptr->polygons[i].vertices.push_back(i * 3u + 1u + 1u);
+    polygon_mesh_ptr->polygons[i].vertices.push_back(i * 3u + 1u);
     // for writing to a file using the library.
-    polygon_mesh_ptr->polygons[i].vertices[0] = (i * 3 + 2);
-    polygon_mesh_ptr->polygons[i].vertices[1] = (i * 3 + 1);
-    polygon_mesh_ptr->polygons[i].vertices[2] = (i * 3 + 0);
+//    polygon_mesh_ptr->polygons[i].vertices[0] = (i * 3 + 2);
+//    polygon_mesh_ptr->polygons[i].vertices[1] = (i * 3 + 1);
+//    polygon_mesh_ptr->polygons[i].vertices[2] = (i * 3 + 0);
   }
 
   ROS_INFO_STREAM(
-    "cloud filled: header: " << mesh_ptr_->cloud.header << "height: " << polygon_mesh_ptr->cloud.height << " width: " << polygon_mesh_ptr->cloud.width << " fields.size: " << polygon_mesh_ptr->cloud.fields.size());
+      "cloud filled: header: " << mesh_ptr_->cloud.header << "height: " << polygon_mesh_ptr->cloud.height << " width: " << polygon_mesh_ptr->cloud.width << " fields.size: " << polygon_mesh_ptr->cloud.fields.size());
 
-ROS_INFO_STREAM("Polygons vector size: " << mesh_ptr_->polygons.size()); }
+  ROS_INFO_STREAM("Polygons vector size: " << mesh_ptr_->polygons.size());
+}
 
 void InfinitamNode::extractITMMeshToPclCloud(
-  const ITMMesh::Triangle* triangleArray,
-  pcl::PointCloud<pcl::PointXYZ>::Ptr point_cloud_pcl) {
-LOG(INFO)<< "extractITMMeshToPclCloud start. ";
-CHECK_NOTNULL(main_engine_);
-CHECK_NOTNULL(&point_cloud_pcl);
-
-std::size_t nr_triangles = 0;
-std::size_t nr_points = 0;
-nr_triangles = main_engine_->GetMesh()->noTotalTriangles;
-nr_points = nr_triangles * 3;
+    const ITMMesh::Triangle& triangleArray,
```

Comment:
I'm always confused when to rename or use our standards in part of software that already exists with other standards. I'll try to do that in those files, that I've written.